pallets/nft/src/benchmarking.rs

```diff
@@ -0,0 +1,124 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+
+use frame_benchmarking::{account, benchmarks};
+use frame_system::RawOrigin;
+
+const SEED: u32 = 0;
+const EMOTE: &str = "RMRK::EMOTE::RMRK1.0.0::0aff6865bed3a66b-VALHELLO-POTION_HEAL-0000000000000001::1F389";
+
+fn create_account<T: Config>(name: &'static str, index: u32) -> T::AccountId {
+	let caller: T::AccountId = account(name, index, SEED);
+	T::Currency::deposit_into_existing(&caller, T::CurrencyBalance::from(1_000_000_u128).into()).unwrap();
+	caller
+}
+
+benchmarks! {
+	create_class {
+		let caller = create_account::<T>("caller", 0);
+		let class_metadata = "just a token class".as_bytes().to_vec();
+		let class_data = "just some class data".as_bytes().to_vec();
+	}: _(RawOrigin::Signed(caller.clone()), class_metadata, class_data, T::CurrencyBalance::from(666_u128).into())
+	verify {
```

Comment:
I would add something to the verify sections of exchange benchmark too. It is helpful when you run tests.